src/components/Main/MainView/ChannelSidebar/ChannelSidebarTopic.vue

```diff
@@ -6,6 +6,7 @@
       @input="onInput"
       @edit-done="onEditDone"
       @edit-start="startEdit"
+      :maxlength="maxlength"
```

Comment:
```suggestion
      :maxlength="200"
```
ここそのまま書いちゃってよさそうです 👍 
(こうすると下の変数なくせると思います)